e2e-tests/common/methods.py

```diff
@@ -33,3 +39,30 @@ def analyzer_supported_entities(data):
         f"{ANALYZER_BASE_URL}/supportedentities?{data}", headers=DEFAULT_HEADERS
     )
     return response.status_code, response.content
+
+
+def image_redactor(file_name: str, color_fill=None):
+    multipart_form_data = get_multipart_form_data(file_name)
+    payload = get_payload(color_fill)
```

Comment:
get_redact_payload, you can inline it as well since its short and will be used only here...

Reply:
it was inline, the method looked very messy. 
I like it this way :)
But you are correct regarding the name and also I made it private.